RickAndMortyFunApp/NetworkServices/NetworkService.swift

```diff
@@ -6,56 +6,97 @@
 //  Copyright © 2019 Rusłan Chamski. All rights reserved.
 //
 
+import Alamofire
 import Foundation
 
 enum NetworkError: Error {
     case invalidURL
 }
 
+enum NetworkRouter: URLRequestConvertible {
```

Comment:
This is RickAndMortyRouter because it's base url is "https://rickandmortyapi.com/api/". Let's change the base url type to `URL` and rename it to baseURL. Also pay attension, that `character` is not a part of base url. Base url is base for all requests. Also, for now let's make it not static. So, `let baseURL = URL(string: "https://rickandmortyapi.com/api/")!` I think it's legal to make it unwrapped optional with `!`. Base url should always be a valid url. If not - we want the app to crash as early as possible